### DISCUSSION TITLE
`import React from "http://self-hosted-verdaccio/react"`

### DISCUSSION THREAD

**Discussion by loynoir** [verdaccio/verdaccio#3523] [Ideas]
**Is your feature request related to a problem?**
Combine self hosted npm registry with self hosted ES Modules registry.

**Describe the solution you'd like**
`import React from "http://self-hosted-verdaccio/react"`

**Describe alternatives you've considered**

**Additional context**
`ije/esm.sh` feat `NPM_CONFIG_REGISTRY=http://self-hosted-esm-sh npm publish foobar`

https://github.com/ije/esm.sh/issues/459

**Comment by juanpicado** [comment]
I'd need to learn more what's esm.sh but maybe you can pitch here what this change might be a good change. For context is something I might be experimenting https://github.com/verdaccio/verdaccio/tree/master/packages/standalone run `node verdaccio.js` as a single file.
